cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_utils.go

```diff
@@ -27,11 +27,11 @@ import (
 	datapackagingv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	"gopkg.in/yaml.v3"
```

Comment:
Shouldn't we unify the versions we use?
[Here](https://github.com/kubeapps/kubeapps/blob/master/pkg/agent/docker_secrets_postrenderer.go#L11) we are using v2.

Reply:
Absolutely agree, that's indeed the purpose of https://github.com/kubeapps/kubeapps/issues/3848.
We have several duplicated deps we ought to simplify sooner or later. Regarding this `v2` vs `v3` I'd clearly go with `v3` since it allows editing the Yaml parsed nodes directly, meaning one is eventually able to create custom nodes manually (e.g., adding commented-out values)